workspace-manager-clienttests/src/main/resources/servers/workspace-wsmtest-resiliency.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "workspace-wsmtest-resiliency",
```

Comment:
This cluster information should be added to the existing `workspace-wsmtest.json` file, instead of creating a new file.

Reply:
Agreed that the information from `workspace-wsmtest-resiliency.json` should merge into `workspace-wsmtest.json`. In `workspace-wsmtest.json`, the test runner service account file is configured as `delegate-user-sa.json` which enables users to bind to the SA for testing. We should enhance the Test Runner library to support a third kind of SA for user delegation, then we will have three SAs: a) a test runner SA for pushing test results and getting cluster metadata, b) a test runner kubernetes SA for resiliency tests, and c) a SA for user delegation for tests that require api authorization. We should open a ticket for this enhancement asap, then we can merge `workspace-wsmtest-resiliency.json` into `workspace-wsmtest.json`.